### ISSUE TITLE
Feature request: vcpkg integrate project that imports vcpkg.targets directly

### ISSUE THREAD

**Issue by davidmatson** [microsoft/vcpkg#31611]
The current implementation of `vcpkg integrate project` is not recommended, because it makes projects hard to share with different users:
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration#linked-nuget-package

Currently, this command generates a NuGet package. However, it could instead take the path to a vcxproj file and edit that file to add

```xml
<Import Project="..\vcpkg\scripts\buildsystems\msbuild\vcpkg.props" />
```

right after Microsoft.Cpp.Default.props and

```xml
<Import Project="..\vcpkg\scripts\buildsystems\msbuild\vcpkg.targets" />
```

right after Microsoft.Cpp.targets.

That implementation of `vcpkg integrate project` would be useful and could be recommended.

**Comment by FrankXie05** [comment]
Please see the [note](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration#linked-nuget-package:~:text=Note,export%20command.).

**Comment by davidmatson** [comment]
@FrankXie05 - yes, I saw this note:

<blockquote>
This approach is not recommended for new projects, since it makes them difficult to share with others. For a portable, self-contained NuGet package, see the [export command](https://learn.microsoft.com/en-us/vcpkg/commands/export).
</blockquote>

Since the current implementation of `vcpkg integrate project` isn't recommended, I'm suggesting an implementation that would be useful and could be recommended, and fits with per-project integration mentioned elsewhere in docs.

**Comment by FrankXie05** [comment]
Do you mean `./vcpkg integrate install`?

**Comment by davidmatson** [comment]
No, I'm suggesting a new implementation of `vcpkg integrate project` that does something more useful and that can be recommended.

**Comment by FrankXie05** [comment]
Sorry for the late reply. :) 
Although adding `Nuget` package integration to aa is a good idea. 
But there are a few things we are considering:
1. `Nuget` is a package manually generated by users using commands. It is difficult for us to identify them, whether it is from the location, information, name of the file, etc.
2. The way to add to vs integration should also take into account the nuget support supported by vs itself.

**Comment by davidmatson** [comment]
@FrankXie05 - I'm not quite understanding the reply.

This issue is a suggestion to provide a non-NuGet mechanism to add vcpkg.props & vcpkg.targets to a vcxproj file.

**Comment by FrankXie05** [comment]
Do you mean to achieve the same integrated function of `./vcpkg integrate project` as `/vcpkg integrate install`?

**Comment by davidmatson** [comment]
No, this would be different from vcpkg integrate install. Per the [docs](https://learn.microsoft.com/en-us/vcpkg/commands/integrate), vcpkg integrate install does:
<blockquote>
vcpkg integrate install
Integrates with [Visual Studio](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration#user-wide-integration) (Windows-only), sets the user-wide vcpkg instance, and displays CMake integration help.
</blockquote>

I believe adding
```xml
<Import Project="..\vcpkg\scripts\buildsystems\msbuild\vcpkg.props" />
```

and

```xml
<Import Project="..\vcpkg\scripts\buildsystems\msbuild\vcpkg.targets" />
```

to the project file is different from per-user installation/integration.
